### PR TITLE
User related examples.

### PR DESCRIPTION
**Pull request by aamirbutt**

### CODE REVIEW

File: feature-examples/python/user/delete_user.py [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+    message = response.json()
+    if message["response"]:
+        if len(message["response"]) != 1:
+            logging.error("DeleteUser failed with response %s", message["response"])
+            raise error_handling.UserException(str(message))
+
+        status = message["response"][0]["status"]
+        if status["code"] == "OK":
+            return True
```
Comment (lgoodby) [inline feature-examples/python/user/delete_user.py:70]: `return`

File: feature-examples/python/user/list_users.py [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+@dataclasses.dataclass(frozen=True)
+class UserData:
+    """User data such as id, name, comment, status of user etc."""
+
+    user_id: int
+    name: str
+    email: str
+    type: str
+    comment: str
+    status: bool
```
Comment (lgoodby) [inline feature-examples/python/user/list_users.py:20]: `str`?

File: feature-examples/python/user/create_user.py [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+
+    response = requests.post(
+        "https://api.vectara.io/v1/manage-user",
+        json=request,
+        verify=True,
+        headers=post_headers,
+        timeout=50,
+    )
+
+    if response.status_code != 200:
```
[the remark this answers is not in the document]
Comment (aamirbutt) [reply feature-examples/python/user/create_user.py:56]: That's a valid concern. But we will have to improve the backend code to return a valid response in such a case.

File: feature-examples/python/user/list_users.py [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+
+    Raises:
+        UserException: In case of any error.
+    """
+    post_headers = {
+        "customer-id": f"{customer_id}",
+        "Authorization": f"Bearer {jwt_token}",
+    }
+
+    # ListUsersType: 2 - Users with customer-level authorization, 3 - All users
```
Comment (lgoodby) [inline feature-examples/python/user/list_users.py:61]: can remove this comment?

**Review by lgoodby — APPROVED** [review]

**Pull request merged**